services/app-api/handlers/reports/unlock.test.ts

```diff
@@ -0,0 +1,94 @@
+import { fetchReport } from "./fetch";
```

Comment:
This file and the implementation can be removed, unless we need them elsewhere (file changed names)

Reply:
removed!